internal/httputil/headers.go

```diff
@@ -17,6 +18,8 @@ const (
 	HeaderPomeriumResponse = "x-pomerium-intercepted-response"
 	// HeaderPomeriumJWTAssertion is the header key containing JWT signed user details.
 	HeaderPomeriumJWTAssertion = "x-pomerium-jwt-assertion"
+	// HeaderPomeriumReProxyPolicy is the header key containing the policy to reproxy a request to.
+	HeaderPomeriumReProxyPolicy = "x-pomerium-reproxy-policy"
```

Comment:
nit: `ReProxy` implies two words but the header itself is `..-reproxy-...` vs `...re-proxy...`; would either change the header name or the variable name for consistently